src/tests/test_utils/common_test_utils/tests/CMakeLists.txt

```diff
@@ -10,7 +10,9 @@ ov_add_test_target(
     DEPENDENCIES
     LINK_LIBRARIES
       common_test_utils
+      func_test_utils
```

Comment:
remove `func_test_utils`?

Reply:
we need this, because `#include "functional_test_utils/common_utils.hpp"` in `openvino/src/tests/functional/shared_test_classes/include/shared_test_classes/base/utils/generate_inputs.hpp`

Reply:
They are linked to `sharedTestClassses`